ontology/yaml/resources/HVAC/entity_types/CHWS.yaml

```diff
@@ -16,6 +16,14 @@
 ### Canonical Types  ###
 ########################
 
+CHWS_RCHWTC_SCHWDT:
+  description: "Chilled water system with temperature control and monitoring on secondary side."
+  is_canonical: true
+  implements:
+  - CHWS
+  - RCHWTC
+  - SCHWDT
```

Comment:
Use `WDT` -- is this really a primary-secondary system? If you are not defining any primary loop fields there is little need to distinguish. If you do have primary loop fields, I suggest bringing them in here.

Reply:
@cstirdivant - apologies no secondary systems, just the primary flow and return to the header. Thanks, Ciaran

Reply:
@Ciaran-Oflannagain thank you for confirming!